src/main/java/com/dev/cinema/dao/impl/CinemaHallDaoImpl.java

```diff
@@ -7,11 +7,14 @@
 import com.dev.cinema.util.HibernateUtil;
 import java.util.List;
 import javax.persistence.criteria.CriteriaQuery;
+import org.apache.log4j.Logger;
 import org.hibernate.Session;
 import org.hibernate.Transaction;
 
 @Dao
 public class CinemaHallDaoImpl implements CinemaHallDao {
+    private static final Logger logger = Logger.getLogger(CinemaHallDao.class);
```

Comment:
```suggestion
    private static final Logger logger = Logger.getLogger(CinemaHallDaoImpl.class);
```
fix this in all the classes